pkg/volume/util.go

```diff
@@ -187,3 +192,47 @@ func GenerateVolumeName(clusterName, pvName string, maxLength int) string {
 	}
 	return prefix + "-" + pvName
 }
+
+// ChooseZone implements our heuristics for choosing a zone for volume creation based on the volume name
+func ChooseZoneForVolume(zones sets.String, pvcName string) string {
```

Comment:
can you document that if anyone creates volumes like foo-index they'll get spreading across zones? (just comment is fine)